src/program/apis/listrr_api.py

```diff
@@ -37,7 +37,7 @@ def __init__(self, api_key: str):
         self.trakt_api = di[TraktAPI]
 
     def validate(self):
-        return self.request_handler.execute(HttpMethod.GET, self.BASE_URL)
+        return self.request_handler.execute(HttpMethod.GET, "")
```

Comment:
we should be validating that the lists are valid with listrr. This not only checks user lists but also ensures that listrr website is live too.